src/graphql/fragments/grants.ts

```diff
@@ -0,0 +1,206 @@
+import { gql } from '@apollo/client'
+
+export const FRAGMENT_BOARD_VOTE_GRANTS = gql`
+  fragment BoardVoteGrantsFragment on BoardVoteGrant {
+    id
+    title
+    name
+    shortDescription
+    description
+    balance
+    status
+    type
+    statuses {
+      status
+      endAt
+      startAt
+    }
+    applicants {
```

Comment:
Do we need to fetch all the applicants here? Where is this used?

(same question for the other queries in this file)

Reply:
No I don't think we need the applicants here, I'll remove it.